app/public/index.html

```diff
@@ -26,6 +26,9 @@
     -->
     <title>React App</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
```

Comment:
good point -> i'll have a look for the actual min versions of the file. I'm assuming you mean make all files local and no CDNs? Bootstrap styles is currently a CDN as well

Reply:
I don't know if you'll see this Robin, but i've not been able to install AND use bootstrap locally. Maybe you could give me a hand? Otherwise we'll have to stick to the cdn for now. Thanks